src/cpp/r/R/Tools.R

```diff
@@ -1134,3 +1134,18 @@ environment(.rs.Env[[".rs.addFunction"]]) <- .rs.Env
    cap <- capabilities(what)
    length(cap) && cap
 })
+
+.rs.addFunction("initTools", function()
+{
+   ostype <- .Platform$OS.type
+   info <- Sys.info()
+   envir <- .rs.toolsEnv()
+   
+   assign(".rs.platform.isUnix",    ostype == "unix",              envir = envir)
```

Comment:
I added these helpers because it's easier to use autocomplete to help with these rather than remember if we're comparing with "Unix" vs. "unix", or "Windows" vs "windows", and so on.